bibtexparser/tests/test_getnames.py

```diff
@@ -0,0 +1,4 @@
+import bibtexparser
+name = "{G{\'{e}}rard} {Ben Arous}"
+print(name)
```

Comment:
Please integrate your test case in the existing `get_names` tests: https://github.com/sciunto-org/python-bibtexparser/blob/master/bibtexparser/tests/test_customization.py#L15

Reply:
on it

Reply:
alright. just tag `@miweiss` me when you're ready for the next github actions run (which I have to manually start).